subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -3141,14 +3248,21 @@ static inline int event_conn_upd_prep(struct ll_conn *conn, uint16_t lazy,
 			cu->interval = conn->llcp_cu.interval;
 			cu->latency = conn->llcp_cu.latency;
 			cu->timeout = conn->llcp_cu.timeout;
+
+			/* hold node rx until the instant's anchor point sync */
+			rx_hold_put(conn, rx->hdr.link, rx);
+
+			if (!IS_ENABLED(CONFIG_BT_CTLR_RX_ENQUEUE_HOLD)) {
+				ll_rx_sched();
+			}
```

Comment:
Referring to my other comment. Here you see that code looks suspicious and one has to start digging, to get a reason on why `ll_rx_sched` is called.

Applies also here: https://github.com/zephyrproject-rtos/zephyr/pull/44791/files#diff-135b49c1c599aa1060145905101574cb58224b39926a1e1a04dda6ff6e8702a0R4857-R4859

Reply:
Updated code to have conditional compiles at the caller compared to being inside callee.